distribution/ui/src/main/resources/PhenoTips/DatabaseFilters.xml

```diff
@@ -83,6 +85,19 @@
 * $services.localization.render('phenotips.databaseFilters.author') {{html clean=false}}&lt;p class="xHint"&gt;$services.localization.render('phenotips.databaseFilters.author.hint')&lt;/p&gt;&lt;input type="text" class="suggestUsers multipleSelection" name="doc.author" value=""/&gt;{{/html}}
 * $services.localization.render('phenotips.databaseFilters.creationDate') #__filters_dateRange("doc.creationDate")
 * $services.localization.render('phenotips.databaseFilters.modificationDate') #__filters_dateRange("doc.date")
+* $services.localization.render('phenotips.databaseFilters.sex')
+  {{html clean=false}}
+    &lt;label&gt;
+      &lt;input type="radio" name="gender" value="" checked="checked" /&gt;
+      $services.localization.render("phenotips.databaseFilters.gender_filter_all")
+    &lt;/label&gt;
+    #foreach($gender in $genderProperty)
+      &lt;label&gt;
+        &lt;input type="radio" name="gender" value="$gender" /&gt;
+        $services.localization.render("phenotips.PatientClass_gender_${gender}")
```

Comment:
The key should be `PhenoTips.PatientClass_gender_${gender}`, which are translated already (except transgender).